menu-memory/memory.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
How does this json get updated between commits? Just locally on your machine?

Reply:
I'm curious about this too

Reply:
It is saved in a lot of the commands to the machine. It will be committed as part of the update call that is ToDo